packages/apollo-engine-reporting/src/treeBuilder.ts

```diff
@@ -6,6 +6,10 @@ import {
 } from 'graphql';
 import { Trace, google } from 'apollo-engine-reporting-protobuf';
 
+function internalError(message: string) {
+  return new Error(`[internal apollo-server error] ${message}`);
```

Comment:
nit: I do think `apollo-engine-reporting` is probably more appropriate, or `internal apollo metrics reporting` if we want to avoid the term engine.

Reply:
I mean that's in the stack trace?